cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         repetition_ids: Optional[Sequence[str]] = None,
         parent_path: Tuple[str, ...] = (),
         extern_keys: FrozenSet['cirq.MeasurementKey'] = frozenset(),
-        use_repetition_ids: bool = True,
+        use_repetition_ids: Optional[bool] = None,
```

Comment:
Nit - consider updating the `use_repetition_ids` docstring below.

Reply:
Updated.